lib/enemyBlueprintTables.js

```diff
@@ -18,22 +18,22 @@ module.exports = function($) {
                 enemies.push(enemy)
             }
 
-            let bpdropchance = $(tr.children[1]).text()
+            let itemdropchance = $(tr.children[1]).text()
 
-            bpdropchance = bpdropchance.slice("Blueprint Drop Chance: ".length, bpdropchance.length - 1)
+            itemdropchance = itemdropchance.slice("Blueprint Drop Chance: ".length, itemdropchance.length - 1)
 
-            enemy = {_id: hash(text), enemyName: text, blueprintDropChance: bpdropchance, mods: []}
+            enemy = {_id: hash(text), enemyName: text, enemyItemDropChance: itemdropchance, items: []}
         }
 
         if(elem.name === "td" && elem.attribs.class !== "blank-row") {
             let chanceElem = tr.children[2]
             let chance = parseChance($(chanceElem).text())
 
-            let modName = $(tr.children[1]).text()
+            let itemName = $(tr.children[1]).text()
 
-            enemy.mods.push({
-                _id: hash(modName),
-                modName: modName,
+            enemy.items.push({
+                _id: hash(itemName),
+                itemName: itemName,
```

Comment:
similarly, here, please maintain passivity, this can break a lot of people's code with little to no notification or time to fix beforehand.

Reply:
So keep both `enemy.mods` and `enemy.items` and add `blueprintDropChance` back?

Reply:
yes